.github/workflows/ci.yaml

```diff
@@ -127,44 +127,59 @@ jobs:
           name: ubuntu-rust-artifacts
       - run: |
           ls -l
-          echo "RELEASE_ARCHIVE=$(cat release-archive.filename)" >>$GITHUB_ENV
+          echo "LINUX_RELEASE_ARCHIVE=$(cat release-archive.filename)" >>$GITHUB_ENV
       - name: Upload release asset
         if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v')
         uses: actions/upload-release-asset@v1
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           upload_url: ${{ steps.create_release.outputs.upload_url }}
-          asset_path: ${{ env.RELEASE_ARCHIVE }}
-          asset_name: ${{ env.RELEASE_ARCHIVE }}
+          asset_path: ${{ env.LINUX_RELEASE_ARCHIVE }}
+          asset_name: ${{ env.LINUX_RELEASE_ARCHIVE }}
           asset_content_type: application/gzip
       - uses: actions/download-artifact@v2
         with:
           name: macos-rust-artifacts
       - run: |
-          echo "RELEASE_ARCHIVE=$(cat release-archive.filename)" >>$GITHUB_ENV
+          echo "MAC_RELEASE_ARCHIVE=$(cat release-archive.filename)" >>$GITHUB_ENV
       - name: Upload release asset
         if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v')
         uses: actions/upload-release-asset@v1
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           upload_url: ${{ steps.create_release.outputs.upload_url }}
-          asset_path: ${{ env.RELEASE_ARCHIVE }}
-          asset_name: ${{ env.RELEASE_ARCHIVE }}
+          asset_path: ${{ env.MAC_RELEASE_ARCHIVE }}
+          asset_name: ${{ env.MAC_RELEASE_ARCHIVE }}
           asset_content_type: application/gzip
       - uses: actions/download-artifact@v2
         with:
           name: windows-rust-artifacts
       - run: |
-          echo "RELEASE_ARCHIVE=$(cat release-archive.filename)" >>$GITHUB_ENV
+          echo "WINDOWS_RELEASE_ARCHIVE=$(cat release-archive.filename)" >>$GITHUB_ENV
       - name: Upload release asset
         if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v')
         uses: actions/upload-release-asset@v1
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           upload_url: ${{ steps.create_release.outputs.upload_url }}
-          asset_path: ${{ env.RELEASE_ARCHIVE }}
-          asset_name: ${{ env.RELEASE_ARCHIVE }}
+          asset_path: ${{ env.WINDOWS_RELEASE_ARCHIVE }}
+          asset_name: ${{ env.WINDOWS_RELEASE_ARCHIVE }}
           asset_content_type: application/zip
+
+      - name: Hook into package registry
+        uses: swift-nav/swift-cli-publisher@v2
+        if: ${{ github.event.ref }} =~ ^refs/tags/v[0-9]+\.[0-9]+\.[0-9]+$
+        env:
+          NAME: "yamllint"
+          PROJECT_SLUG: "swift-nav/yamllint-bin"
+          DL_LINUX: ${{ env.LINUX_RELEASE_ARCHIVE }}
+          DL_MAC: ${{ env.MAC_RELEASE_ARCHIVE }}
+          DL_WIN: ${{ env.WINDOWS_RELEASE_ARCHIVE }}
+          TOOLS: "yamllint"
```

Comment:
https://github.com/swift-nav/package-registry/blob/aaed9303c54fd3420a513ae875f6bcea96366b61/data/yamllint#L1

would make tools: `yamllint,null,null,null,null`
is there a reason for the nulls/ do we have to follow that pattern

also is linked via web DL not github DL

Reply:
We don't need the nulls, probably there from a really old version of Swift CLI that had a fixed size array for the tools